maquetado/index.html

```diff
@@ -10,5 +10,39 @@
 </head>
 <body>
   <h2 class="title">Front-End Tools</h2>
+  <div class="technologies">
+    <div class="content-center">
+      <div class="technology">
+        <img class="logo" src="./assets/react_icon.svg" alt="React">
```

Comment:
I would change this class name to `tech-logo` since you could have more than once logo on your screen